Makefile

```diff
@@ -100,47 +124,82 @@ all: run ## Alias for 'run'.
 include build/*.mk
 
 start-docker: ## Starts the docker containers for local development.
+ifeq ($(IS_CI),false)
```

Comment:
Should this just use docker-compose?

Reply:
we can, but i would say to wait a bit more :) propose to do in a followup

Reply:
Yup. Do in followup.